product_variant_tree_qty_available/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Product Variant Tree View Qty Available",
+    "summary": "Adds the 'QTY' button to product variant tree view",
+    "version": "14.0.1.0.1",
+    "category": "Inventory",
+    "website": "https://github.com/OCA/product-variant",
+    "author": "Ooops, Cetmix, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
Incoherent licensing between this entry and headers.

Reply:
Thanks,I'll fix it